packages/transaction-log/package.json

```diff
@@ -60,6 +60,7 @@
     "@types/systemjs": "^6.1.1",
     "@types/webpack-env": "^1.16.2",
     "axios": "^1.7.2",
+    "moment": "^2.30.1",
```

Comment:
moment.js appears to be a bit outdated. Can you check out date-fns?

This is optional.

Reply:
I used moment because it was the Lib that the previous version of the OpenHIM used. I can migrate over to date-fns since on the moment.js page it says it is in legacy mode.